styles.css

```
@@ -11534,6 +11534,21 @@ body:has(#side-panel.active) #close-fullscreen-cover-btn {
     #fullscreen-cover-overlay .fs-volume-bar {
         width: 100%;
     }
+
+    #fullscreen-cover-overlay .fullscreen-top-actions {
```

Stylelint (17.14.0)

[error] Duplicate selector "`#fullscreen-cover-overlay` .fullscreen-top-actions", first used at line 11241 (no-duplicate-selectors)

(no-duplicate-selectors)